### DISCUSSION TITLE
Uncaught Error: Class 'CConfigGeneralHelper' not found

### DISCUSSION THREAD

**Discussion by boybka** [BGmot/zabbix#42] [Q&A]
Hi guy,

I install 2FA for zabbix 6.4.13, but i have a issue. Can you help me fix it?

[Fri Apr 12 08:39:31.584304 2024] [php7:error] [pid 19084] [client 192.168.226.1:53207] PHP Fatal error:  Uncaught Error: Class 'CConfigGeneralHelper' not found in /usr/share/zabbix/include/classes/helpers/CTwofaHelper.php:6\nStack trace:\n#0 /usr/share/zabbix/include/classes/core/CAutoloader.php(77): require()\n#1 [internal function]: CAutoloader->loadClass()\n#2 /usr/share/zabbix/app/controllers/CControllerTwofaEdit.php(52): spl_autoload_call()\n#3 /usr/share/zabbix/include/classes/mvc/CController.php(480): CControllerTwofaEdit->doAction()\n#4 /usr/share/zabbix/include/classes/core/ZBase.php(638): CController->run()\n#5 /usr/share/zabbix/include/classes/core/ZBase.php(233): ZBase->processRequest()\n#6 /usr/share/zabbix/include/config.inc.php(25): ZBase->run()\n#7 /usr/share/zabbix/zabbix.php(22): require_once('/usr/share/zabb...')\n#8 {main}\n  thrown in /usr/share/zabbix/include/classes/helpers/CTwofaHelper.php on line 6, referer: http://192.168.226.128/zabbix/zabbix.php?action=authentication.edit

**Comment by BGmot** [comment]
I confirm the issue. Do you need this to be fixed for 6.4.13 or it’s ok to release for 6.4.14?
Please create an issue here https://github.com/BGmot/zabbix/issues

**Comment by BGmot** [comment]
Actually fixed. Just follow procedure again please and it will update all the files to correct ones.
